apps/video/index.html

```diff
@@ -33,9 +33,11 @@
 
     <!-- Web Components -->
     <script defer src="shared/elements/config.js"></script>
+    <script defer src="shared/js/component_utils.js"></script>
```

Comment:
I agree, we will be working on this in the future as we unify the web component architecture.